tests/test_directive.py

```diff
@@ -26,3 +26,32 @@ def test_get_gh_contributors_repository_not_exists(self):
 
         contributors = _get_gh_contributors('dgarcia360', 'reponotfound')
         assert contributors == []
+
+    def test_render_contributors_list_sorting_by_contributions(self):
+
+        contributors = [{'login': 'dgarcia360', 'contributions': 3, 'htmlurl': 'http://#'},
+                        {'login': 'premsa', 'contributions': 2, 'htmlurl': 'http://#'},
+                        {'login': 'user', 'contributions': 1, 'htmlurl': 'http://#'}]
+        rendered_contributors_list = _render_contributors_list(contributors)
+
+        assert rendered_contributors_list.astext() == 'dgarcia360 - 3 contributions\n\npremsa - 2 contributions\n\nuser - 1 contributions'
```

Comment:
This test is not passing, should be: 

"1 contribution" instead of "1 contributions"